fgd/point/point/point_spotlight.fgd

```diff
@@ -23,8 +23,8 @@
 	// Inputs
 	input LightOn(void) : "Turn the spotlight on."
 	input LightOff(void) : "Turn the spotlight off"
-	input SetColor[since_ASW, GMod](color255) : "Change the color of the spotlight."
-	input ForceUpdate[since_ASW, GMod](void) : "Force an update of the spotlight position and orientation."
```

Comment:
I worry about stuff like this -- does removing `since_ASW` here but not `GMod` mean that this is now only available for GMod?

Reply:
Starting to think `since_ASW` should probably remain, it's technically valid for our use case (GO is `since_ASW`)

Reply:
That's true. I split them up like this for simplicity (gmod gets removed here later). If `since_ASW` existed then we'd have to keep ASW as a game, whereas my methodology was to remove *everything* not in Chaos to have a less complicated base.

Reply:
Okay I didn't know `GMod` gets removed in a later PR yet so that's fine with me then.